js/private/image/index.ts

```diff
@@ -28,15 +29,46 @@ type Entries = { [path: string]: Entry }
 
 type Compression = 'gzip' | 'none'
 
-function findKeyByValue(entries: Entries, value: string): string {
+function findKeyByValue(entries: Entries, value: string): string | undefined {
     for (const [key, { dest: val }] of Object.entries(entries)) {
         if (val == value) {
             return key
         }
     }
-    throw new Error(
-        `couldn't map ${value} to a path. please file a bug at https://github.com/aspect-build/rules_js/issues/new/choose`
-    )
+    return undefined
+}
+
+async function isSymbolicLink(p: string) {
```

Comment:
This returns more then a boolean so shouldn't have the `is*` naming. Can this just be `readlink` or `readBazelLink` or something like that?